README.md

```diff
@@ -1,2 +1,42 @@
-# Project-1-Mobile
- 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project 1-Mobile .
+
+## Built With
+
+- CSS
+- HTML
+
+## Live Demo
+
+[Live Demo Link](https://victoromart.github.io/Project-1-Mobile/)
+
+
+
+
+## Authors
+
+👤 **Victor Omar Toraño**
+
+- GitHub: [@VictorOmarT](https://github.com/VictorOmarT)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/victoromartm/)
+
+
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] You should update your README file. You don't have an MIT license in your repository. You can check this [website](https://www.freecodecamp.org/news/how-open-source-licenses-work-and-how-to-add-them-to-your-projects-34310c3cf94/) for how you add it. If you won't add please delete the license section.
- [ ] Also please update or delete acknowledgments section. These words from template.